### ISSUE TITLE
Less swipe length

### ISSUE THREAD

**Issue by karandashek** [zefoy/ngx-swiper-wrapper#192]
Hi @sconix! 
I'm, trying to find "sweet point" for swipe responsiveness, but not sure how change swipe length to be less than default. I checked Swiper api docs and found some configs like "shortSwipes, longSwipes, longSwipesRatio etc", but it seems there is nothing changes based on them. I tried research and found that when i'm swiping (with "FollowFinger = true"), i need swipe too long before i see swiper started following the finger, i think it takes ~50px swipe length to see swiper started follow. I thought this may be related with application performance and checked provided example (https://zefoy.github.io/ngx-swiper-wrapper/) and found the same case. 
I'm not sure is this issue or just i can't find correct config variable, but if compare "follow finger" in https://zefoy.github.io/ngx-swiper-wrapper/ and 
http://idangero.us/swiper/demos/010-default.html 
it seems it's possible to follow finger "more immediately" :). 
Basically, is there way to decrease/minimize this "follow finger" delay?

Thanks!

**Comment by karandashek** [comment]
Thanks for response!
ok, it seems i was not very clear with all that long-read:)
If you'll check this example of ngx-swiper-wrapper https://zefoy.github.io/ngx-swiper-wrapper/ and try to swipe slow(actually there is no difference fast/slow, swipe length is the same, but in slow case it's easier to see what i mean) you will see some distance between you started swipe and slide followed your finger. Is there any way to remove/reduce this distance to follow finger earlier? 
Just i mentioned original Swiper as parent library which can follow finger much earlier, i think if i'll ask Swiper developers they will provide example which we already know can follow finger earlier and will ask me to ask you :)

**Comment by sconix** [comment]
This is just a wrapper, so this library only passes in configuration to Swiper. So swiping etc. is handled by Swiper.

**Comment by karandashek** [comment]
ok, got it! Thank you!
